bower.json

```diff
@@ -9,9 +9,15 @@
     "testing"
   ],
   "devDependencies": {
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "webcomponentsjs-v1": "webcomponents/webcomponentsjs#v1"
+    "web-component-tester": "6.0.0-prerelease.5",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#v1"
```

Comment:
That sounds like the right thing to do? The [bower docs page about the dependencies field](https://github.com/bower/spec/blob/59c8f0e8f8444cbdd71a091919d28d0fa29d56fe/json.md#dependencies) doesn't seem to mention anything about being able to include version numbers when using a git url. :/ I'll try it out in a bit to see if it's there but undocumented.